lib/client.js

```diff
@@ -738,10 +802,13 @@ class Client {
     if (lastModified) res.setHeader('Last-Modified', lastModified);
   }
 
-  // Download file
+  // Download file generating a file attachment.
+  // If cannot read file, `404 Not Found`
+  // client error status response code is sent
   //   filePath <string> file to download
-  //   attachmentName <string> (optional) name to save downloaded file
-  //   callback <Function>
+  //   attachmentName <string> name to save downloaded file,
+  //      optional, default: basename of filePath
```

Comment:
In metadoc's format four space indentation on subsequent lines is used to break the line:
```suggestion
  //       optional, default: basename of filePath
```